### ISSUE TITLE
December Update Broke Windower

### ISSUE THREAD

**Issue by SG1234567** [EasyFarm/EasyFarm#150]
I'm sure this has happened in the past with previous updates as I just wanted to let you know. I wanted to get a ticket in to document that it broke your program as well.

**Comment by Mykezero** [comment]
I've created a new build with the newest version of fface.dll. See if that fixes the problem!

**Comment by shadow9555** [comment]
Just gave it a try. Even ran the program as Administrator nothing seems to be happening

**Comment by Mykezero** [comment]
Well shuckles; I'll have to give it a try when I get home. Thanks for giving it a shot ^^;

**Comment by frenchies5** [comment]
I tried it as well and nothing happened.

**Comment by Mykezero** [comment]
Just to be clear, the program does open but it doesn't work properly after hitting the start button?

**Comment by frenchies5** [comment]
That is correct.  You can load and select character. But when you start the program it doesn't do anything. 

Sorry, trolling your post. I use your program daily and I can't thank you enough!

**Comment by SG1234567** [comment]
Follow works, routes works, but monster targeting, and self targeting (casting a spell when it wears off) does not work. 

Widower has put in a good amount of updates, and someone said that there were a lot of memory position changes made in this update.

Thanks for the quick feedback, your program is much appreciated.

**Comment by novio** [comment]
Indeed. There's an issue open for the targeting dilemma on fface but I think it might actually be ffacetools that needs an update.

Have you checked out elitemmo api by any chance? I'm not a fantastic programmer but I've managed a couple of small 'utility' programs with it. It would probably be a breeze for someone like you to adopt and likely easier to manage vs fface

**Comment by Mykezero** [comment]
@novio Thanks for the information. I've started adding support for elitemmo as a memory readining wrapper in a different branch. Hopefully FFACE is updated again soon with some more fixes since it will take a while to get the elitemmo wrapper as stable as the fface one. 

The new code is [here](https://github.com/EasyFarm/EasyFarm/tree/memory-api) if you did want to play around with it. Sadly, it's too buggy for release at this point as far as pathing is concerned. The good news is at least everything else is works ^^;

**Comment by Mykezero** [comment]
@novio That version has also been built as well here: 
https://ci.appveyor.com/project/Mykezero/easyfarm/build/artifacts

The current memory reader can be configured by changing the EasyFarm.exe.config file by uncommenting the specific reader you want to use. It's set to use MMOElite api as the default for this version so it shouldn't have to be adjusted. 

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <appSettings>
    <add key="MemoryWrapper" value="EasyFarm.EliteMMOWrapper"/>
    <!--<add key="MemoryWrapper" value="EasyFarm.FFACEWrapper"/>-->
  </appSettings>
  ...
</configuration>
```

**Comment by frenchies5** [comment]
I don't know if this helps but I have tested the fface.dll on another
program and it works fine.

**Comment by Mykezero** [comment]
@frenchies5  Which program are you using?

**Comment by frenchies5** [comment]
Gamebot. I'm not sure if its the same packets or whatnot

**Comment by jonii99** [comment]
still not work,can't find object

**Comment by jonii99** [comment]
old version work,

**Comment by frenchies5** [comment]
Which version please.

**Comment by jonii99** [comment]
i dont know. old version with new fface.dll , it is work, target mob working

**Comment by jonii99** [comment]
old version is 1.4.1.11

**Comment by frenchies5** [comment]
Could you possibly compress it and load it on here?

**Comment by link1456** [comment]
The link that Mykezero posted in this issue box thing seems to work for me, the one with the Elitemmo wrapper thing. Seems to assist my main and cast blizz when i want.

 Thanks for everything.

**Comment by Mykezero** [comment]
@frenchies5 If the other versions are not working for you, I can recompile and make available version 1.4.1.11 if you'd like?
